resources/views/layout/metadata.blade.php

```diff
@@ -28,6 +28,24 @@
 <meta name="keywords" content="osu, peppy, ouendan, elite, beat, agents, ds, windows, game, taiko, tatsujin, simulator, sim, xna, ddr, beatmania, osu!, osume">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+@if (isset($opengraph))
+    @if (isset($opengraph['section']))
+        <meta property="og:site_name" content="{{ 'osu! » '.$opengraph['section'] }}">
+    @else
+        <meta property="og:site_name" content="{{ 'osu!` }}">
+    @endif
+
+    <meta property="og:type" content="website">
+
+    @if (isset($canonicalUrl))
```

Comment:
this is not optional so it should be part of the check above (or no check at all so whoever adding opengraph to pages will notice this)